TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -221,6 +238,7 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
         _ symbolSearchViewController: TradeItSymbolSearchViewController,
         didSelectSymbol selectedSymbol: String
     ) {
+        self.selectedSymbolChanged = true
```

Comment:
maybe we make this `self.selectedSymbolChanged = self.order.symbol != selectedSymbol`? IMO it fits better definition wise.

This controller flow is a little confusing as well -- not sure we can do much about it right now, but it requires knowing a thing or two about when different actions are called

Reply:
Do we reset this to `false` after we use it?

Reply:
Agreed that this is getting confusing.